client/src/Backend/Miner/MinerManager.ts

```diff
@@ -5,6 +5,7 @@ import _ from 'lodash';
 import { ChunkStore } from '../../_types/darkforest/api/ChunkStoreTypes';
 import { HashConfig, MinerWorkerMessage } from '../../_types/global/GlobalTypes';
 import { getChunkKey } from './ChunkUtils';
+import DefaultWorker from './miner.worker.ts?worker';
```

Comment:
why are there question marks in the imports now?

Answered my own q, storing here for reference: https://vitejs.dev/guide/assets.html#explicit-url-imports

Reply:
Yep! Vite has a bit of docs to understand. Fwiw, we were doing something similar with RegExps + webpack loaders. This is just built-in with Vite!